Makefile

```diff
@@ -91,7 +91,7 @@ test-pydocstyle:
 
 .PHONY: test-pylint
 test-pylint:
-	pylint craft_cli
+	pylint craft_cli --disable=cyclic-import
```

Comment:
Why is this needed? Is it reporting a false positive?

Reply:
Yes. Note that import order in `craft_cli/__init__.py` is defined to avoid the cyclic import. But pylint falls in "the trap" anyway...

Reply:
If you need to toggle a check, do it here instead https://github.com/canonical/craft-cli/blob/main/.pylintrc#L72

Reply:
ack, improved.